infra/terraform/modules/auth0/outputs.tf

```diff
@@ -0,0 +1,3 @@
+output "aws_client_id" {
```

Comment:
I think we want all clients' OIDC client ID as outputs so they can be easily grabbed from `terraform output`, or passed into other modules like `federated_identity`.

Reply:
Yep